tests/python/test_script_methods.py

```diff
@@ -81,12 +81,15 @@ def test_run_script_multi(use_cluster, context):
     c = Client(None, use_cluster, logger_name=context)
     c.put_tensor("srpt-multi-out-data-1", data)
     c.put_tensor("srpt-multi-out-data-2", data_2)
-    c.set_function("two-to-one", two_to_one)
-    c.run_script(
+    c.set_function_multigpu("two-to-one", two_to_one, 0, 2)
+    c.run_script_multigpu(
         "two-to-one",
         "two_to_one",
         ["srpt-multi-out-data-1", "srpt-multi-out-data-2"],
         ["srpt-multi-out-output"],
+        0,
+        0,
+        2
```

Comment:
I don't understand this change

Reply:
the original test for muli gpu here was using run_script instead of run_script_multigpu. the offset, num of gpus and first gpu is required positional arguments for the run_script_gpu func. 

I also might be misunderstanding your Q: I put the arguments as following bc
offset: 0 (start with MPI rank 0)
first_gpu: 0 (start with gpu 1 since its zero-based)
num_gpus: 2 (im using a spider node which has 10 gpus but I thought I would just put 2 since others might not)

does this ans your Q? Or should these not be hardcoded?